src/Storage/Storage.Management/StorageAccount/NewAzureStorageAccount.cs

```diff
@@ -286,6 +286,7 @@ public bool EnableAzureActiveDirectoryDomainServicesForFile
         }
         private bool? enableAzureActiveDirectoryDomainServicesForFile = null;
 
+        [CmdletParameterBreakingChange("EnableLargeFileShare", ChangeDescription = "EnableLargeFileShare parameter is being deprecated")]
```

Comment:
How about:
"EnableLargeFileShare parameter will be deprecated in a future release."

Reply:
Done